guides/source/working_with_javascript_in_rails.md

```diff
@@ -41,12 +41,14 @@ Rails ships with CoffeeScript by default, and so the rest of the examples
 in this guide will be in CoffeeScript. All of these lessons, of course, apply
```

Comment:
Should probably change this note :)

Reply:
@connorshea Do you want those two lines to be removed?

Reply:
Yes.

Reply:
You can likely remove this entire paragraph. We ship with ES6 by default, and since ES6 is intended to supersede ES5, it probably isn't worth distinguishing.